kratos/solving_strategies/builder_and_solvers/p_multigrid/noop_constraint_assembler.hpp

```diff
@@ -0,0 +1,61 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Máté Kelemen
+//
+
+#pragma once
+
+// Project Includes
+#include "solving_strategies/builder_and_solvers/p_multigrid/constraint_assembler.hpp" // ConstraintAssembler
+#include "includes/kratos_parameters.h" // Parameters
+
+
+namespace Kratos {
+
+
+template <class TSparse, class TDense>
+class NoOpConstraintAssembler final : public ConstraintAssembler<TSparse,TDense>
```

Comment:
Ok, now you are trying to get into my region of making hard to understand names :O...., and missing doxygen.